### ISSUE TITLE
S3 Support - add ACL option

### ISSUE THREAD

**Issue by mimamuh** [directus/docker#37]
Hi,

right now the [`api.php`](https://github.com/directus/docker/blob/master/projects/api/api.php#L101) maps environment variables to the config but misses an option for `ACL` in case I wanna use the `aws` storage provider. 

By the way:
To use `aws` as storage provider it still misses the `aws/aws-sdk-php` package preinstalled in the docker image, see here #36. But I created the image my own because of that and changed them a little bit. But for the future rewrite it would be cool if the `ACL` option is supported through a env-var in the [`api.php`](https://github.com/directus/docker/blob/master/projects/api/api.php#L101) too like so:

```php
'storage' => [
    // ...
    'options' => [
        'ACL' => getenvs('STORAGE_ACL', 'public-read'),
    ],
],
```

**Comment by mimamuh** [comment]
Or probably it would be better to have basic `config/api.php` configuration file customizeable through env-vars, like it is now for the most important stuff. But also support a `config/custom-api.php` to overwrite configs with a custom file. Otherwise each possible option needs a env-var which may get too complicated. Better would be when I would be able to mount my `custom-api.php` into the container and it will be merged with the `api.php` config somehow. In case I need a more specific custom config like for the use-case above, I can do it without relying on creating my own image or similar.

**Comment by mvhirsch** [comment]
@mimamuh you can! Just add a file to `/var/www/html/config/docker/api.php` and it will get loaded for you instead. Take a look here: https://github.com/directus/docker/blob/master/projects/api/api.php#L4-L7

I'll provide an PR for the options, since I need them too.

**Comment by mvhirsch** [comment]
@WoLfulus I'm not exactly sure, how this should be configured (but I think `public-read` is now a default). Could you please help us on this?

Guessing this _should_ be the correct configuration afaik, using env:

```
DIRECTUS_STORAGE_OPTIONS_ACL = 'public-read'
```

**Comment by WoLfulus** [comment]
@mvhirsch Yes, that should do it on newer images, but I'll still confirm it anyways this week.

**Comment by helge000** [comment]
@WoLfulus - Seeing this issue still open, 'public-read' is not a sensible default:

```
DIRECTUS_STORAGE_OPTIONS_ACL = 'public-read'
```

It should really be unset by default - as in this case the ACL of the bucket is used. The way it is now it will not work out of the box with default private buckets.
